lxd/api_1.0.go

```diff
@@ -15,6 +15,7 @@ import (
 	clusterConfig "github.com/canonical/lxd/lxd/cluster/config"
```

Comment:
commit message needs to be updated to reflect the suggestion in the API extension description.